src/diffusers/pipelines/flux/pipeline_flux_controlnet.py

```diff
@@ -748,9 +748,11 @@ def __call__(
             )
 
             # set control mode
+            orig_mode_type = type(control_mode)
```

Comment:
I think we should accept both `int` or a list (of length 1) `int`
```suggestion
            if isinstance(control_mode, list):
                control_mode = [control_mode]
            if len(control_mode) > 1:
                raise ValueError(" For `FluxControlNet`, `control_mode` should be an `int` or a list contain 1 `int`") 
```

Reply:
I think you meant to do the opposite here? i.e. if `not isinstance(control_mode, list)` then convert `control_mode` into a singleton list?

Reply:
oops! yes!